uber/tests/test_custom_tags.py

```diff
@@ -106,3 +106,18 @@ def test_linebreaksbr(self, test_input, expected):
     ])
     def test_normalize_newlines(self, test_input, expected):
         assert expected == normalize_newlines(test_input)
+
+
+class TestUrlToLink(object):
```

Comment:
I couldn't figure out this error when I was running my unit tests:
```
============================================================================================================ FAILURES =============================================================================================================
_________________________________________________________________ TestUrlToLink.test_urltolink[/regular/url-url_kwargs1-<a href="/regular/url">/regular/url</a>] __________________________________________________________________

self = <uber.tests.test_custom_tags.TestUrlToLink object at 0x7f47caec75c0>, url_args = '/regular/url', url_kwargs = {}, expected = '<a href="/regular/url">/regular/url</a>'

    @pytest.mark.parametrize('url_args, url_kwargs, expected', [
        ('', {}, ''),
        ('/regular/url', {}, '<a href="/regular/url">/regular/url</a>'),
        (['/regular/url', 'normaltext'], {}, '<a href="/regular/url">normaltext</a>'),
        (['/regular/url', 'normaltext', '_blank'], {}, '<a href="/regular/url" target="_blank">normaltext</a>'),
        (['&<>"\'', 'normaltext'], {}, '<a href="&amp;&lt;&gt;&#34;&#39;">normaltext</a>'),
        (['/regular/url', '&<>"\''], {}, '<a href="/regular/url">&amp;&lt;&gt;&#34;&#39;</a>'),
        (['/regular/url', 'normaltext', '&<>"\''], {}, '<a href="/regular/url" target="&amp;&lt;&gt;&#34;&#39;">normaltext</a>')
    ])
    def test_urltolink(self, url_args, url_kwargs, expected):
>       assert expected == url_to_link(*url_args, **url_kwargs)
E       TypeError: url_to_link() takes from 0 to 3 positional arguments but 12 were given
```

The tests themselves seem to work, as I had several fail on me while I was figuring out exactly how our escaping worked (turns out we use `&#34;` for double-quotes instead of the more commonly-seen `&quot;`!). I must have set all this up wrong, though.